packages/ckb-indexer/tests/cell_collector.unit.test.ts

```diff
@@ -208,3 +232,133 @@ test("validateQueryOption#validate scriptLenRange", (t) => {
   });
   t.is(error2.message, "scriptLenRange[1] must be a hexadecimal!");
 });
+
+test("validateQueryOption#validate should support multiple queries", (t) => {
+  t.throws(
+    () => {
+      new CellCollector(indexer, [{ lock: lockScript }, {}]);
+    },
+    undefined,
+    "throw error if lock and query both not provided in queryOption[1]"
+  );
+
+  t.throws(
+    () => {
+      new CellCollector(indexer, [{ lock: lockScript }, { type: "empty" }]);
+    },
+    undefined,
+    "throw error if lock is not provided and type is empty in queryOption[1]"
+  );
```

Comment:
why `{ type: "empty" }` will be throw here?

Reply:
https://github.com/ckb-js/lumos/pull/475/files#diff-f9c531afc5c4264761e548055e5ab3b3b6b33ce8754fdac1fdb196c2e7212155R129

But it is copied from here. I just reuse previous code.

Reply:
https://github.com/IronLu233/lumos/blob/c49e0bc3b191cc4aeb53487581eee7df2f646f18/packages/ckb-indexer/tests/cell_collector.unit.test.ts#L127